#161342817 Display bookmarked articles

victor-nweze-deel opened the pull request:
#### What does this PR do?
Display a users bookmarked articles

#### Description of Task to be completed?
- Create component for rendering bookmarks
- Make api calls to retrieve user bookmarked articles
- Write tests for all functionality

#### How should this be manually tested?
- Git clone this repository
- Cd into the `haven-ah-frontend` folder
- Run `npm run start:dev` to start the development server
- Login and hit profile from the menu bar

#### What are the relevant pivotal tracker stories?
- [#161342817](https://www.pivotaltracker.com/story/show/161342817)

#### Screenshots (if appropriate)
<img width="1248" alt="screen shot 2018-12-20 at 4 25 49 pm" src="https://user-images.githubusercontent.com/41582565/50293809-b6bbf700-0474-11e9-884e-76eba96e9b03.png">

Commit:
feat(profile): get user profile

- create a component to display the user profile

- write css to style the user profile component

- write actions, reducers, sagas for data manipulation

- write test for feature

ucheg6 commented on src/components/views/ProfilePage.jsx:
```diff
@@ -0,0 +1,181 @@
+});
+
+ProfilePage.propTypes = {
+  userProfileAction: PropTypes.func.isRequired,
```
Great Job Victor. Please add default props for props that are not required.

victor-nweze-deel replied:
Thanks @ucheg6

victor-nweze-deel replied:
lol 🤣

victor-nweze-deel committed:
ft(profile): Display a users bookmarked articles

- Create component for rendering bookmarks
- Make api calls to retrieve user bookmarked articles
- Write tests for all functionality
